delivery_transsmart/models/stock_picking.py

```diff
@@ -131,13 +140,13 @@ def _transsmart_document_from_stock_picking(self):
             "AddressCountry": self.partner_id.country_id.code or '',
             "ColliInformation": [
```

Comment:
Just collapse the [ and { together, as wll as the } and ], and you can save two lines and an indentation level